backend/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint .",
+    "start": "nodemon ./routers/index.js"
```

Comment:
We use the "start" script in production, and in production we won’t use nodemon, only during development. 
Please rewrite the script for nodemon to “start:dev” or something similar. 
“start" should just start the app in production. ("start": "node <serverfile>”) ﻿ ﻿ ﻿
Please include both scripts! ﻿